packages/framework/block-std/src/view/element/lit-host.ts

```diff
@@ -27,13 +29,17 @@ import type { BlockSpec, SpecStore } from '../../spec/index.js';
 import type { ViewStore } from '../view-store.js';
 
 import { BlockStdScope } from '../../scope/index.js';
+import { PropTypes, requiredProperties } from '../decorators/required.js';
 import { RangeManager } from '../utils/range-manager.js';
 import { WithDisposable } from '../utils/with-disposable.js';
 import { ShadowlessElement } from './shadowless-element.js';
 
 export const docContext = createContext<Doc>('doc');
 export const stdContext = createContext<BlockStdScope>('std');
 
+@requiredProperties({
+  specs: PropTypes.arrayOf(PropTypes.object),
```

Comment:
What about doc and std?

Reply:
Done. Fixed circular dependencies between `view.js` and `scope.js`.